Contributors.md

```diff
@@ -0,0 +1,7 @@
+
+
+
+carlos-sg
```

Comment:
We don't use a contributors file, we rely on Git's history for a list of contributors, and use Developer Certificate of Origin which requires sign off using a real name and email address to trace where code comes from.